africa/morocco/README.md

```diff
@@ -0,0 +1 @@
+[marrakech](marrakech.md)
```

Comment:
Add a title to this page, so the user knows in which country they are looking at.
```suggestion
# Morocco

[marrakech](marrakech.md)
```